pkg/webreg/testdata/zz_fixture_TestWorkflowDotFile_step_to_chain.yaml

```diff
@@ -0,0 +1,46 @@
+digraph Webreg {
```

Comment:
You're right.  Fixed with:

```diff
--- pkg/webreg/graphviz.go
+++ pkg/webreg/graphviz.go
@@ -184,7 +184,7 @@ func (b *graphBuilder) addEdge(i int) {
 		if b.edge.srcType != noEdge {
 			b.graph.edges = append(b.graph.edges, b.edge)
 		}
-		b.edge = edge{srcType: nodeType, srcNode: b.edge.dstNode}
+		b.edge = edge{srcType: nodeType, srcNode: b.edge.dstNode, dstType: nodeType}
 }
 
 func ChainGraph(name string, chains registry.ChainByName) ([]byte, error) {
```

Note that the files were still generated correctly (which is why I didn't notice this problem), since we only differentiate between `subgraphType` and "anything else".